modules/_labs/core/react/lib/theming/useTheme.ts

```diff
@@ -17,12 +17,13 @@ import {defaultCanvasTheme} from './theme';
  * Tracked on https://github.com/emotion-js/emotion/issues/1193.
  */
 export function useTheme(theme?: Object): CanvasTheme {
+  const context = React.useContext(ThemeContext);
```

Comment:
This will have to be an exception, at least how `useTheme` is written because it can be used both inside and outside a Hook context. Ideally this behavior is a different function for that use-case, but this change will break functionality.

Reply:
Perhaps @anicholls can add more insight

Reply:
updated